### DISCUSSION TITLE
GET `/warehouses/fulfillments` fails with 302 "You are being redirected"

### DISCUSSION THREAD

**Discussion by goldenneilm** [Sellbrite/Sellbrite-API#30] [Q&A] [answered]
Beginning on 2/15/24, calls to GET /warehouses/fulfillments begain to fail with 302, for example

`GET https://api.sellbrite.com:443/v1/warehouses/fulfillments/{UUID}?sb_status=open&sb_payment_status=all&shipment_status=none&min_ordered_at=2024-02-10T00%3A00%3A00Z&page=1&page_size=100`

Which had previously succeeded, began to fail with 302 "You are being redirected"

```
<?xml version="1.0"?> 
<html> 
<body>You are being <a href="https://api.sellbrite.com/merchants/sign_in">redirected</a>.
</body> 
</html>
```

- It appears to be an auth error, however other requests with the same credentials succeed, for example `GET /warehouses` returns the expected response data
- The first occurence of the error was 02/15/2024 06:04:27 EST (`x-request-id: 7d660615-cc89-48db-8209-fb38526e2964`). It was isolated at first; subsequent attempts succeeded.
- Since then it increased in frequency, and as of 02/16/2024 12:20:01 EST (`x-request-id: 687d15be-2587-4b1d-b940-fbfb9a065805`) the request fails 100% of the time.

This error is preventing us from syncing fulfillments data to the warehouse, please urgently advise.

**Comment by rengler-godaddy** [comment, marked as answer]
Hello - Thanks for reaching out!

It appears this was due to an update that our team rolled back this morning. Can you let us know if you’re still seeing the same issue?

**Reply by goldenneilm** [reply]
Confirmed, this error has not reoccurred since 02/20/2024 14:50:09 EST. Thank you SellBrite team
